# 3D vector of translation.
float64[3] translation
# Rotation in scaled angle-axis (Lie algebra) form.
float64[3] rotation
